Feature/desktop initialization

mm-pazouki opened the pull request:
In this branch, I added a desktop version of my portfolio

changes include:

responsive desktop version for the intro section
responsive desktop version for the Projects section
responsive desktop version for the about section
responsive desktop version for the contact form
new footer

mm-pazouki committed:
1- Add Images from figma
2- Create Desktop version for Intro/Projects
3- Add desktop support for about section
4- Add desktop for contact form
5- Add footer
6- fix stylelint
7- Fix webhind errors
8- Add rel="noopener" for <a> tag

mm-pazouki committed:
Fix Stylelint errors

Zuhaib042 commented on Contents/Index-desktop.css:
```diff
@@ -0,0 +1,331 @@
+@media screen and (min-width: 600px) {
```
- [x] Kindly consider changing this `min-width` value from 600px to 768px as the convention over desktop screen size begins from 768px width.
requirement
  - Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).

Zuhaib042 commented on Contents/Index-desktop.css:
```diff
@@ -0,0 +1,331 @@
+    font-size: 1.6rem;
+    display: flex;
+    flex-direction: row;
+    align-content: center;
```
- [x] Kindly consider using `Poppins` font from google as we need to follow figma guidelines in fonts, margins, paddings, texts and colors.
here is the [Poppins](https://fonts.google.com/specimen/Poppins?query=poppi) font link.
kindly consider using Poppins font for all the texts where in figma Poppins is used.
- [x] And also kindly consider using the exact font-weight and font-size for nav links mentioned in the figma following design guidelines.
- [x] You may also want to remove the bottom outline for the same purpose. our design should be identical to the figma design.

mm-pazouki committed:
Online Demo added

Zuhaib042 commented on Index.html:
```diff
@@ -1,269 +1,176 @@
+      </ul>
+    </nav>
+  </header>
+  <section id="intro">
+    <div>
+      <h1>Hi, I'm <b>Mohammad Mehdi Pazouki</b>, Happy to see you</h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Take a look of my works. If you like what you see
+        and have a project you need coded, don't hesitate and contact me.
+      </p>
+      <button type="button" class="dark small">Start collaboration</button>
+      <div id="logo"></div>
+    </div>
+  </section>
```
- [x] Kindly consider giving your elements exact margins, paddings, font-sizes and font-weights as it can be seen in the figma design because we need to follow it as per requirements. kindly have a look at the positioning of your elements and positioning of elements in figma design. kindly consider fixing this elements positioning issue.
  - Also for example the font size of the p tag is bigger in your design so consider fixing it.
your deisgn 
![r](https://user-images.githubusercontent.com/96683843/236262870-8bde7d2d-4af2-4b52-a5a8-872650d6d5b6.jpg)

figma design
![f](https://user-images.githubusercontent.com/96683843/236262929-1b725f64-77dc-4395-805c-3e25c997f5dd.jpg)

Zuhaib042 commented on Index.html:
```diff
@@ -1,269 +1,176 @@
+      <div id="logo"></div>
+    </div>
+  </section>
+  <section id="Projects">
```
- [x] There are some issues in this project section. kindly consider fixing them following figma guidelines.
   - The main heading's font-size of the project in the figma is greater than the size in your design.
   - The padding between technologies in your design is lower than in figma design.
   - the width and background-color of `see project` button in both cards are different in your design.
- [x] There are also three regular projects cards with laptop images in figma design that don't exist in your design so please consider adding these 3 project cards at exactly at the same position as in the figma design.
 your design
![a](https://user-images.githubusercontent.com/96683843/236266162-38659f04-8199-42f0-a508-907348677f0f.jpg)
![v](https://user-images.githubusercontent.com/96683843/236267481-810ceb58-d66e-4830-a809-2bd451260266.jpg)

figma design
![s](https://user-images.githubusercontent.com/96683843/236266182-34260514-c51a-48e4-8069-b5a218e41c23.jpg)
![z](https://user-images.githubusercontent.com/96683843/236267530-b568fea5-5653-4082-846e-7aa9ab6f983c.jpg)

Zuhaib042 commented on Index.html:
```diff
@@ -1,269 +1,176 @@
+    </div>
+  </section>
+  <section id="Projects">
+    <h1>Projects</h1>
```
- [x] You are taking multiple `h1`s in your project. we only use one `h1` on each webpage  for main heading in hero section following best semantic and SEO practices. kindly consider changing this and all other instances of `h1` except the main heading in hero section to `h2...h6`.

Zuhaib042 commented on Index.html:
```diff
@@ -1,269 +1,176 @@
+  <section id="Projects">
+    <h1>Projects</h1>
+  </section>
+  <section id="about">
+    <div id="about-info">
+      <h1>About</h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Taka a look of my works. If you like what you see
+        and have a project you need coded, don't hesitate and contact me.
+      </p>
+    </div>
+    <button type="button" class="dark small">Get my CV &#10515;</button>
+    <ul id="skills-and-frameworks">
+      <li>Languages
+        <ul id="Languages">
+          <li>JavaScript</li>
+          <li>Python</li>
+          <li>HTML</li>
+          <li>CSS</li>
         </ul>
```
- [ ] There are some issues in this about section. kindly consider fixing them following figma guidelines.
   -  There is not enough padding at left and right side of about section. kindly consider increasing the padding on higher displays, for example greater than 1200px.
   - The About heading font is different and smaller in your design.
   - The p tag font-size is bigger in your design.
   - And the padding of `Get my CV` button is lower in your design so consider increasing it a bit.
your design
![n](https://user-images.githubusercontent.com/96683843/236271027-87460595-4550-4d88-a065-f6425657c43b.jpg)

figma design
![c](https://user-images.githubusercontent.com/96683843/236271046-f989c706-6e90-45d8-80a9-191340c53918.jpg)

Zuhaib042 commented on Index.html:
```diff
@@ -1,269 +1,176 @@
+      </li>
+    </ul>
+  </section>
+  <form action="https://formspree.io/f/xeqwnkjg" method="POST" id="contact-form">
+    <h1>Get Started</h1>
+    <p>I prefer to speak by my gestgures.
+    </p>
+    <fieldset>
+      <label for="customer-name">
+        Your name <span class="errorText"></span>
+        <br />
+        <input id="customer-name" name="customer-name" type="text" placeholder="Full name" maxlength="30" required />
+      </label>
+    </fieldset>
+    <fieldset>
+      <label for="customer-email">
+        Your email <span class="errorText"></span>
+        <br />
+        <input id="customer-email" name="customer-email" type="email" placeholder="something@domain.com" required />
+      </label>
+    </fieldset>
+    <span class="errorText"></span>
+    <textarea id="customer-message" name="customer-message" maxlength="500" placeholder="Enter your message here..."
+      required></textarea>
+    <span class="errorText" id="submitError"></span>
+    <button id="btn-submit" class="dark small" type="button" name="submit1">Start collaboration</button>
+  </form>
```
- [x] There are some issues in this contact section. kindly consider fixing them following figma guidelines.
   - The Get Started heading font is different in your design.
   - The p tag font-size is bigger in your design.
   - And the padding of `Start collaboration` button is different in your design so consider making it identical to figma design.
 
![n](https://user-images.githubusercontent.com/96683843/236271707-ec38b845-6d7b-4b7f-8615-f21551f344e9.jpg)

your design
![b](https://user-images.githubusercontent.com/96683843/236271727-1d9b9d58-9099-4ab1-84b7-e28c98d26f2e.jpg)

Zuhaib042 commented on Index.html:
```diff
@@ -1,269 +1,176 @@
+    <span class="errorText" id="submitError"></span>
+    <button id="btn-submit" class="dark small" type="button" name="submit1">Start collaboration</button>
+  </form>
+  <section id="footer">
+    <a href="https://www.linkedin.com/in/mohammad-mehdi-pazouki/" target="_blank" rel="noopener">Get my CV &#10515;</a>
+    <ul>
```
- [x] There are some issues in this footer. kindly consider fixing them following figma guidelines.
    - The `Get my CV` link should be aligned  to left side and the social media links to the right as per figma design.
    - There is some white space beneath your footer that doesn't exist in figma design so consider removing that white space.
your design

![y](https://user-images.githubusercontent.com/96683843/236272753-dc9b8389-63e4-4d32-bf62-7f4a4fb5ecdb.jpg)

figma design
![m](https://user-images.githubusercontent.com/96683843/236272637-c13ccdbe-0614-4de3-a0d7-4e923bd6c04b.jpg)

Zuhaib042 reviewed (changes requested):
Hi @mm-pazouki 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://media.giphy.com/media/VlqzeaRnhCvPLfrSh6/giphy.gif)

## Highlights
✅ Linters passing
✅ Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mm-pazouki committed:
Button size changed
footer size changed
footer white space is loom extention bug
font size changed
font family fixed
button color fixed
button size fixed.

mm-pazouki committed:
Stylelint fix

divinecharlotte commented on Contents/index.css:
```diff
@@ -1,1514 +1,450 @@
-html,
```
- [x] Kindly as requested by the previous reviewer consider using the same font family as the one used in Figma design to make your work look as expected

divinecharlotte commented on Index.html:
```diff
@@ -1,269 +1,176 @@
+    </div>
+  </section>
+  <section id="Projects">
+    <h1>Projects</h1>
```
- [x] Kindly as requested by the previous reviewer consider making sure that you stick to the Figma design as much as possible mainly here am talking about your mobile version consider making sure that as you develop the desktop version make sure that the mobile version is not messed up  that will make your work more professional
- check on this part 
your design 👇 
<img width="427" alt="Screen Shot 2023-05-04 at 21 33 21" src="https://user-images.githubusercontent.com/60146030/236311607-1b14487c-1b65-4e8a-b2e3-7034d1af742c.png">



Figma design 👇 
<img width="465" alt="Screen Shot 2023-05-04 at 21 33 54" src="https://user-images.githubusercontent.com/60146030/236311786-400f4a41-0a2f-4ffa-a427-25e376a36d5b.png">


consider doing so on the desktop version as well

- your design 👇 
<img width="1344" alt="Screen Shot 2023-05-04 at 21 47 35" src="https://user-images.githubusercontent.com/60146030/236314940-481fe57c-7ba8-4712-9bc2-84cc45768aa4.png">

- Figma design 👇 

<img width="867" alt="Screen Shot 2023-05-04 at 21 49 15" src="https://user-images.githubusercontent.com/60146030/236315106-cbdf67f0-06f7-40a5-9c87-0f54eb9e28be.png">

mahbubul14 replied:
> * [x]  Kindly as requested by the previous reviewer consider making sure that you stick to the Figma design as much as possible mainly here am talking about your mobile version consider making sure that as you develop the desktop version make sure that the mobile version is not messed up  that will make your work more professional
> * check on this part
>   your design point_down
> 
> <img alt="Screen Shot 2023-05-04 at 21 33 21" width="427" src="https://user-images.githubusercontent.com/60146030/236311607-1b14487c-1b65-4e8a-b2e3-7034d1af742c.png">
> 
> Figma design point_down <img alt="Screen Shot 2023-05-04 at 21 33 54" width="465" src="https://user-images.githubusercontent.com/60146030/236311786-400f4a41-0a2f-4ffa-a427-25e376a36d5b.png">
> 
> Consider doing so on the desktop version as well
> 
> * your design point_down
> 
> <img alt="Screen Shot 2023-05-04 at 21 47 35" width="1344" src="https://user-images.githubusercontent.com/60146030/236314940-481fe57c-7ba8-4712-9bc2-84cc45768aa4.png">
> 
> * Figma design point_down
> 
> <img alt="Screen Shot 2023-05-04 at 21 49 15" width="867" src="https://user-images.githubusercontent.com/60146030/236315106-cbdf67f0-06f7-40a5-9c87-0f54eb9e28be.png">

- [ ] As the previous reviewers suggested, Kindly try to match your design to the Figma design. :+1:

divinecharlotte commented on Index.html:
```diff
@@ -1,269 +1,176 @@
+    </ul>
+  </section>
+  <form action="https://formspree.io/f/xeqwnkjg" method="POST" id="contact-form">
+    <h1>Get Started</h1>
+    <p>I prefer to speak by my gestgures.
+    </p>
+    <fieldset>
+      <label for="customer-name">
+        Your name <span class="errorText"></span>
+        <br />
+        <input id="customer-name" name="customer-name" type="text" placeholder="Full name" maxlength="30" required />
+      </label>
+    </fieldset>
+    <fieldset>
+      <label for="customer-email">
+        Your email <span class="errorText"></span>
+        <br />
+        <input id="customer-email" name="customer-email" type="email" placeholder="something@domain.com" required />
+      </label>
+    </fieldset>
+    <span class="errorText"></span>
+    <textarea id="customer-message" name="customer-message" maxlength="500" placeholder="Enter your message here..."
+      required></textarea>
+    <span class="errorText" id="submitError"></span>
+    <button id="btn-submit" class="dark small" type="button" name="submit1">Start collaboration</button>
+  </form>
```
- [x]  Kindly as requested by the previous reviewer consider making sure that you stick to the Figma design as much as possible mainly here am talking about your mobile version consider making sure that as you develop the desktop version make sure that the mobile version is not messed up that will make your work more professional
check on this part
your design 👇
<img width="321" alt="Screen Shot 2023-05-04 at 21 39 18" src="https://user-images.githubusercontent.com/60146030/236312532-c61fde1b-b351-4e4d-a1e8-df9b14060e9d.png">

figma design 👇 
<img width="291" alt="Screen Shot 2023-05-04 at 21 42 03" src="https://user-images.githubusercontent.com/60146030/236312734-42077980-cddd-45dd-bf21-a6d168705665.png">

mahbubul14 replied:
- [x] As the previous reviewers suggested, the "Get Started" and the details font should be matched with the Figma design, and also try to match the get my resume section as the Figma design.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign](https://user-images.githubusercontent.com/70483835/236413635-b33d770e-cc8f-4479-956e-e72a44f1ab3b.png) | ![getstarted_figma](https://user-images.githubusercontent.com/70483835/236413571-753c7c5f-a1eb-45f2-a7b8-16ec112632b5.png)

divinecharlotte reviewed (changes requested):
Hi @mm-pazouki     , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- Descriptive Pr 👍
- linters are passing 👍 




 
![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)
## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

mm-pazouki committed:
Margin Fixed
Font Fixed

mm-pazouki committed:
Linter

mm-pazouki committed:
Linter

mm-pazouki committed:
LinterStyle

mm-pazouki committed:
Linter

mahbubul14 reviewed (changes requested):
Hi @mm-pazouki,

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [x] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [x] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

mm-pazouki committed:
Update index.css

OyePriscilla commented on Index.html:
```diff
@@ -1,269 +1,176 @@
+      </li>
+    </ul>
+  </section>
+  <form action="https://formspree.io/f/xeqwnkjg" method="POST" id="contact-form">
+    <h1>Get Started</h1>
+    <p>I prefer to speak by my gestgures.
+    </p>
+    <fieldset>
+      <label for="customer-name">
+        Your name <span class="errorText"></span>
+        <br />
+        <input id="customer-name" name="customer-name" type="text" placeholder="Full name" maxlength="30" required />
+      </label>
+    </fieldset>
+    <fieldset>
+      <label for="customer-email">
+        Your email <span class="errorText"></span>
+        <br />
+        <input id="customer-email" name="customer-email" type="email" placeholder="something@domain.com" required />
+      </label>
+    </fieldset>
+    <span class="errorText"></span>
+    <textarea id="customer-message" name="customer-message" maxlength="500" placeholder="Enter your message here..."
+      required></textarea>
+    <span class="errorText" id="submitError"></span>
+    <button id="btn-submit" class="dark small" type="button" name="submit1">Start collaboration</button>
+  </form>
```
- [ ] As pointed out by the previous reviewer, kindly design the form according to the given design. 🙏 

> |     | Given template                                | Your Design                                   |     |
> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
> |     | <img width="120" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/236432890-a52e9b7f-99d2-4091-9d0b-f69e25daf1c1.png"> |<img width="270" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/236432547-71659753-17fb-42cc-9516-35e8b0a5a15d.png"> |

mm-pazouki replied:
All of you are stupid, do you understand it is a desktop version branch and I could not work on the mobile version here? and still, review the mobile version????????

TuyishimireEric replied:
### Hello @mm-pazouki 👋 👋,
I understand your frustration, but please remember to remain respectful when communicating with others. Please, keep the conversation professional and respectful. 🙏 

Note that the milestone requires the webpage to be fully responsive. This means that it should work seamlessly on both desktop and mobile devices. Therefore, it is important to address any issues related to the mobile version as well. 😎😎

OyePriscilla reviewed (changes requested):
Hi @mm-pazouki 👏 👏, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so-close-so-close-seth-meyers](https://user-images.githubusercontent.com/103386183/196425724-8ca5825c-b02a-4983-b976-4bd0a2f70f9a.gif)

 ## Highlights
Use media queries to make app responsive ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

TuyishimireEric reviewed (approved):
# Status: APPROVED 🟢  
### Hi  @mm-pazouki  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
![done](https://media.giphy.com/media/l1uga8ObPBnIBoXUQ/giphy.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Descriptive PR👍
✔ Responsive web design👍
### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
